server/src/main/java/org/opensearch/node/Node.java

```diff
@@ -1101,6 +1106,7 @@ protected Node(
                 b.bind(MetaStateService.class).toInstance(metaStateService);
                 b.bind(PersistedClusterStateService.class).toInstance(lucenePersistedStateFactory);
                 b.bind(IndicesService.class).toInstance(indicesService);
+                b.bind(RemoteStoreStatsTrackerFactory.class).toInstance(remoteStoreStatsTrackerFactory);
```

Comment:
Should this be behind the remote store feature flag?

Reply:
Same issue as above